chain/jsonrpc/src/lib.rs

```diff
@@ -958,6 +968,23 @@ impl JsonRpcHandler {
     }
 }
 
+#[cfg(feature = "sandbox")]
+impl JsonRpcHandler {
+    async fn sandbox_patch_state(
+        &self,
+        patch_state_request: near_jsonrpc_primitives::types::sandbox::RpcSandboxPatchStateRequest,
+    ) -> Result<Value, RpcError> {
+        actix::spawn(
+            self.client_addr
+                .send(NetworkClientMessages::Sandbox(NetworkSandboxMessage::SandboxPatchState(
+                    patch_state_request.records,
+                )))
+                .map(|_| ()),
+        );
+        Ok(Value::String("".to_string()))
```

Comment:
Following the example of all the refactored JSON RPC methods, we should return some normal structure ([even if it is an empty struct](https://github.com/near/nearcore/blob/2c0c4973ea3b6730c40654b6299d599e2631e6d8/chain/jsonrpc-primitives/src/types/status.rs#L10-L11)), and then serialize it to the Value in `process_request` (in the future, we will write a macro attribute to avoid repetitive code in the `process_request`, but meanwhile, it is the contract we want to maintain in order to make things cleaner.

P.S. Empty string is a really unhelpful response (null is not helpful either, but it indicates that there is nothing to be reported back while string might be any string).

Reply:
I see, good to know!